### ISSUE TITLE
[Bug] Signup from gmail : Saves Full name as First Name in the profile page

### ISSUE THREAD

**Issue by anupammajhi** [jsecademy/www#3]
Signup from gmail : Saves Full name as First Name in the profile page.
It should be dividing the name into First name and Last name just like the profile page is supposed to as per current design.

**Comment by rick4470** [comment]
Nope! it's the other way around. It should be asking for just first name not full name. Thanks for reporting this.

**Comment by anupammajhi** [comment]
I didn't mean the "sign up form"
I am talking about the the signup via gmail.
Once you reach the profile page after signing up, it registers the full name as first name.

Example Image:

![image](https://cloud.githubusercontent.com/assets/8258915/23273923/e96ee130-fa26-11e6-99bf-8b1bbd84e576.png)
